tds/src/main/java/thredds/server/wfs/WFSController.java

```diff
@@ -55,19 +54,22 @@ public void httpHandler(HttpServletRequest hsreq, HttpServletResponse hsres) {
 			
 			// 
 			if(request != null) {
+				WFSLogger.logInfo("WFS Server received GetCapabilities request.");
 				if(request.equals(WFSRequestType.GetCapabilities.toString())) {
 					getCapabilities(wr, hsreq);
 				}
 			}
 			
 			else{
-				wr.append("ncWFS on THREDDS - ");
-				wr.append(hsreq.getScheme() + "://" + hsreq.getServerName() + ":" + hsreq.getServerPort() + "/thredds/wfs");
+				WFSExceptionWriter owsExcept = new WFSExceptionWriter("WFS server error. REQUEST parameter is required.", "request", "MissingParameterValue", hsres);
+				owsExcept.write();
+				return;
 			}
 		}
 		
 		catch(IOException io) {
-			
+			WFSLogger.logError("ERROR: retrieval of writer failed.");

```

Comment:
Could this be switched to use a native SLF4J Logger object? Doing so would permit logging the exception's stack trace (and any underlying exceptions' messages and stack traces) by passing the exception in the arguments. Example: `Logger.error("my custom message", io)`;

Reply:
Mentioned in another comment that the wrapped WFSLogger was to provide a central log for all wfs related problems and information. Smaller more granular logs could be used, but the problem can also be solved by allowing for an Exception to be passed as a second argument. Regardless SLF4J stuff will be all removed in the time being in consideration of your comment below.

Reply:
:+1: